test/e2e_node/topology_manager_test.go

```diff
@@ -205,6 +218,8 @@ func configureTopologyManagerInKubelet(f *framework.Framework, oldCfg *kubeletco
 	// Set the Topology Manager policy
 	newCfg.TopologyManagerPolicy = policy
 
+	newCfg.TopologyManagerScope = scope
```

Comment:
Can you double check this bc I see in the failing test:
```
/b/f/w/test/e2e_node/topology_manager_test.go:221:8: newCfg.TopologyManagerScope undefined (type *"k8s.io/kubernetes/pkg/kubelet/apis/config".KubeletConfiguration has no field or method TopologyManagerScope)
```

see: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/93653/pull-kubernetes-bazel-build/1293105913422417922

Reply:
Thank you for pointing this out. This will fail as long as [https://github.com/kubernetes/kubernetes/pull/92967](PR) is not merged.